tests/_async/test_mapping.py

```diff
@@ -0,0 +1,223 @@
+#  Licensed to Elasticsearch B.V. under one or more contributor
+#  license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright
+#  ownership. Elasticsearch B.V. licenses this file to you under
+#  the Apache License, Version 2.0 (the "License"); you may
+#  not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+import json
+
+from elasticsearch_dsl import Keyword, Nested, Text, analysis
+from elasticsearch_dsl._async import mapping
```

Comment:
Yes, there's a few. I used a search/replace approach to convert the code, so I kept these as close as the originals as I could. But I agree, I'll remove these, because if we use them here people will copy them and that's not good.